debian/changelog

```diff
@@ -0,0 +1,5 @@
+matrix-synapse (0.33.9) bionic; urgency=low
```

Comment:
can you give it a different name to distinguish it from the py2 packaging please.

Reply:
done